modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,244 @@
+= APIkit for GraphQL Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configurations
+---
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| API definition | String |  |  | *x*
+| Introspection Enabled | Boolean |  Enables schema introspection. |  false | 
+| Max Query Depth Allowed | Number |  The maximum depth a query can have. The value must be greater than 1. |  100 | 
+| Max Query Complexity Allowed | Number | The maximum number of data fields a query can include. The value must be greater than 1. |  100 | 
+| Name | String |  The identifier of this element is used to reference it in other components. |  | *x*
+|===
+
+==== Associated Operations
+
+* <<Route>> 
+* <<Serialize>> 
+* <<SerializeBatch>> 
+
+==== Associated Sources
+
+* <<DataFetcher>> 
+* <<DataLoader>> 
+
+== Operations
+
+=== Route
+
+`<graphql-router:route>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*
+| Content | Any |  |  #[payload] | 
+| Output MIME Type | String |  The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding | String |  The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Streaming Strategy  a| 
+* <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configure if repeatable streams should be used and their behavior. |  | 
+| Target Variable | String |  The name of a variable on which the operation output is placed. |  | 
+| Target Value | String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* <<Config>> 
```

Comment:
Should this link to the "=== Config" title?
If so, we link to content with an xref: xref:filename.adoc#articletitle[title or phrase to explain title action]

Reply:
Yes

Reply:
OK! Thanks!